[2024-11-10T15:28:31.215+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-11-10T15:28:31.246+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_pipeline_dag_with_duration.merge_with_orders manual__2024-11-10T15:21:57.162279+00:00 [queued]>
[2024-11-10T15:28:31.263+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_pipeline_dag_with_duration.merge_with_orders manual__2024-11-10T15:21:57.162279+00:00 [queued]>
[2024-11-10T15:28:31.265+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 2
[2024-11-10T15:28:31.297+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): merge_with_orders> on 2024-11-10 15:21:57.162279+00:00
[2024-11-10T15:28:31.329+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=534) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-11-10T15:28:31.331+0000] {standard_task_runner.py:72} INFO - Started process 536 to run task
[2024-11-10T15:28:31.319+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'etl_pipeline_dag_with_duration', 'merge_with_orders', 'manual__2024-11-10T15:21:57.162279+00:00', '--job-id', '144', '--raw', '--subdir', 'DAGS_FOLDER/etl_pipeline_dag.py', '--cfg-path', '/tmp/tmp_rasrmg4']
[2024-11-10T15:28:31.333+0000] {standard_task_runner.py:105} INFO - Job 144: Subtask merge_with_orders
[2024-11-10T15:28:31.427+0000] {task_command.py:467} INFO - Running <TaskInstance: etl_pipeline_dag_with_duration.merge_with_orders manual__2024-11-10T15:21:57.162279+00:00 [running]> on host 1669fe1f842b
[2024-11-10T15:28:31.611+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_pipeline_dag_with_duration' AIRFLOW_CTX_TASK_ID='merge_with_orders' AIRFLOW_CTX_EXECUTION_DATE='2024-11-10T15:21:57.162279+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-11-10T15:21:57.162279+00:00'
[2024-11-10T15:28:31.613+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2024-11-10T15:28:31.613+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2024-11-10T15:28:31.614+0000] {logging_mixin.py:190} INFO - Current task name:merge_with_orders state:running start_date:2024-11-10 15:28:31.246886+00:00
[2024-11-10T15:28:31.615+0000] {logging_mixin.py:190} INFO - Dag name:etl_pipeline_dag_with_duration and current dag run status:running
[2024-11-10T15:28:31.615+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-11-10T15:28:31.616+0000] {logging_mixin.py:190} INFO - Simulating task for 6 seconds.
[2024-11-10T15:28:37.624+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-11-10T15:29:00.870+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-11-10T15:29:00.885+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=etl_pipeline_dag_with_duration, task_id=merge_with_orders, run_id=manual__2024-11-10T15:21:57.162279+00:00, execution_date=20241110T152157, start_date=20241110T152831, end_date=20241110T152900
[2024-11-10T15:29:00.888+0000] {job.py:229} INFO - Heartbeat recovered after 29.80 seconds
[2024-11-10T15:29:00.954+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2024-11-10T15:29:00.962+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2024-11-10T15:29:00.968+0000] {logging_mixin.py:190} INFO - Dag name:etl_pipeline_dag_with_duration queued_at:2024-11-10 15:21:57.300030+00:00
[2024-11-10T15:29:00.970+0000] {logging_mixin.py:190} INFO - Task hostname:1669fe1f842b operator:PythonOperator
[2024-11-10T15:29:01.008+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-11-10T15:29:01.106+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-11-10T15:29:01.198+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
